tests/PHPStan/Rules/Classes/data/bug-12281.php

```diff
@@ -0,0 +1,10 @@
+<?php // lint >= 8.2
+
+namespace Bug12281;
+
+#[\AllowDynamicProperties]
+readonly class BlogData { /* … */ }
+
+/** @readonly  */
+#[\AllowDynamicProperties]
+class BlogDataPhpdoc { /* … */ }
```

Comment:
not sure we want the same error for phpdoc readonly, as we have for native readonly...?

Reply:
I wouldn't do that for PHPDoc `@readonly`.